src/test/java/lotto/domain/LottoGameTest.java

```diff
@@ -0,0 +1,59 @@
+package lotto.domain;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatIllegalArgumentException;
+
+public class LottoGameTest {
+    private Lotto winningNumbers;
+    private Lotto lottoNumbers;
+
+    @BeforeEach
+    public void setup() {
+        lottoNumbers = new Lotto(Arrays.asList(new LottoNumber(1),
+                new LottoNumber(2),
+                new LottoNumber(3),
+                new LottoNumber(4),
+                new LottoNumber(5),
+                new LottoNumber(6)));
```

Comment:
생성자의 수가 늘어날수록 클래스를 사용하기 위한 유연성은 높아집니다.

```suggestion
        lottoNumbers = new Lotto(1, 2, 3, 4, 5, 6);
```